pages/index.js

```diff
@@ -87,6 +88,9 @@ export default function Home() {
         } else {
             setSourceCode(preinstalled_programs.basic.mandelbrot);
             setDataFetch(true);
+            if(urlParams.size>0){
+                openNotification("Invalid Parameter Found.", "bottomRight");
```

Comment:
I would do something more descriptive, perhaps this:
```suggestion
                openNotification("The URL contains an invalid parameter.", "bottomRight");
```

Reply:
Sure, That would be great.